tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +829,70 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string   $expected
+     * @param string[] $properties
+     */
+    public function testSkipProperties($className, $propertyName, $expected, $properties)
+    {
+        $proxy = $this->generateProxy($className);
+        $ghostObject = new $proxy(
```

Comment:
I suggest adding an additional assertion that verifies that the proxy is not initialized at the end of the test